lib/__tests__/test.js

```diff
@@ -4,6 +4,36 @@ const fs = require('fs');
 const path = require('path');
 const {walkDir} = require('../execution-helper');
 const rewriter = require('../rewriter');
+const {exec} = require('child_process');
+
+const integrationInput = fs.readFileSync(
+    './lib/__testfixtures__/static-methods.input.mjs', 'utf8'
+);
+const integrationOutput = fs.readFileSync(
+    './lib/__testfixtures__/static-methods.output.mjs', 'utf8'
+);
+
+/**
+ * Runs a test on the rewriter script's file-overwrite
+ */
+async function runIntegrationTest() {
+  const path = './integration.js';
+  // write input to file
+  fs.writeFileSync(path, integrationInput, 'utf8');
+
+  // rewrite file
+  await exec(`./bin/rewrite.js ${path}`);
+
+  const fileOutput = fs.readFileSync(path, 'utf8');
+
+  // check against expected output
+  test('the rewriter command works as expected', () => {
+    expect(fileOutput).toEqual(integrationOutput);
+  });
+
+  // delete file
+  fs.unlinkSync(path);
```

Comment:
This should be in a `finally {}` block so that even if the test fails it still gets deleted.